concepts/basics/about.md

```diff
@@ -46,3 +46,4 @@ Car.new.run
 Finally, bear in mind that the `Integer` object holds values that may be defined as one or more (consecutive) digits and its methods support many of the [mathematical operators][integers-docs].
 
 [integers-docs]: https://ruby-doc.org/core-2.7.0/Integer.html
+
```

Comment:
This is displaying what is an <EOL> character to terminate the line 48. Since this is a markdown file and intended to be treated as a text file, it should have an end of line terminator, otherwise when building documentation, this will do unexpected things, (such as concatentate the next document as a continuation of this line, unless additional work is done to monitor close of file and opening of the other file.

Newline or EOL at the end of the last line of text files saves future headaches.